README.md

```diff
@@ -38,6 +38,8 @@ mustache meta.yml ../templates/default.nix.mustache > default.nix
 mustache meta.yml ../templates/README.md.mustache > README.md
 mustache meta.yml ../templates/.travis.yml.mustache > .travis.yml
 mustache meta.yml ../templates/coq.opam.mustache > coq-aac-tactics.opam
+mkdir -p .github/workflows
+mustache meta.yml ../templates/coq-action.yml.mustache > .github/workflows/coq-action.yml
```

Comment:
Maybe it is time to just remove this list of commands. Especially as they might not be really accurate since I'm not sure that AAC-Tactics will contain a GitHub Action, and will probably never contain both a GitHub Action and a Travis file.

Reply:
OK if you want, but in this case, we could also remove the `.travis.yml` line, shouldn't we?

Reply:
But you're right that that paragraph specifically deals with AAC-Tactics; so I will remove the coq-action command and replace it with just a remark